src/decorators/README.md

```diff
@@ -0,0 +1,28 @@
+# Decorators
+
+## Overview
+
+Decorators provide annotations for class methods and properties. Decorators use the form `@decorator` where `decorator` is the name of the function that will be called at runtime. 
+
+## Basic Usage
+
+### txIdFromHeader
+
+This simple decorator allows you to annotate a Controller operation. The decorator will annotate the operation with the value of the header `X-Transaction-Id` from the request. 
```

Comment:
"a Controller operation parameter" or "a Controller method argument"

also

"annotate the operation parameter" or "annotate the method argument"

(use the same variant in both places please)